bin/run-test.sh

```diff
@@ -50,3 +50,15 @@ fi
 
 # Run non-interference tests
 node test/non-interference/http-e2e.js || exit 1
+
+
+# If on travis, and this is not a pull request build, run the system-tests
+# as well.
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
```

Comment:
This will probably fail locally as the travis env variable isn't set but the key + iv variables aren't either.